components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -376,6 +376,13 @@ private fun HitResult.isLink(): Boolean =
     ((this is HitResult.UNKNOWN && src.isNotEmpty()) || this is HitResult.IMAGE_SRC) &&
         getLink().startsWith("http")
 
+private fun HitResult.isLinkForOtherThanWebpage(): Boolean {
+    val link = getLink()
+    return ((this is HitResult.UNKNOWN && src.isNotEmpty()) || this is HitResult.IMAGE_SRC) &&
+        link.startsWith("http") &&
+        (link.endsWith("html") || link.endsWith("htm"))
```

Comment:
nit: I this logic should be negated as we want to show menus on everything, except `urls` that end in `html` extensions.

Reply:
Totally. Thanks!